### PR TITLE
Upgrade command of homebrew installs minikube

### LINKED ISSUE #18213: Issue with k8s.io/de/docs/tasks/tools/install-minikube/
**This is a Bug Report**

**Problem:**
The command "brew cask install minikube" does not work anymore. 

**Proposed Solution:**
Now it is only possible to install minikube with this command: "brew install minikube"
**Page to Update:**
https://kubernetes.io/de/docs/tasks/tools/install-minikube/

### PR DESCRIPTION
**Pull request by gavinfish**
Fix #18213.

### CODE REVIEW

**Comment by sftim** [comment]
I don't use Homebrew, so I'll unassign myself.
/unassign

**Comment by oke-py** [comment]
Updates has already been merged in #17711 
Please wait for the merge into master.

File: content/ja/docs/tasks/tools/install-minikube.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ MinikubeはVMの中ではなくホスト上のKubernetesのコンポーネント
 [Homebrew](https://brew.sh)を使うことでmacOSにMinikubeを簡単にインストールできます。
 
 ```shell
-brew cask install minikube
```
Comment (inductor) [inline content/ja/docs/tasks/tools/install-minikube.md]: Please remove this change as it's already dealt with in the Japanese translation branch.

**Comment by inductor** [comment]
@gavinfish This commit `2b58022eff990e58642f777f7285ecfc697dc4f6` will cause conflict... Could you possibly re-commit without the change?

**Comment by inductor** [comment]
I guess the German/Spanish translation team should take a look at this

**Comment by electrocucaracha** [comment]
This is related to this [change](https://github.com/Homebrew/homebrew-core/pull/45603)

```bash
$ brew search minikube                                                                                                                                                                                                                                                                                          
==> Formulae
minikube ✔

If you meant "minikube" specifically:
It was migrated from homebrew/cask to homebrew/core.
```

/lgtm

**Comment by gavinfish** [comment]
Any updates here?

**Comment by inductor** [comment]
@kubernetes/sig-docs-es-reviews
@kubernetes/sig-docs-de-reviews

ping

**Comment by gavinfish** [comment]
/assign electrocucaracha
for no response for two weeks

**Comment by inductor** [comment]
I guess @alexbrand needs to approve, too? :thinking:

**Comment by inductor** [comment]
any @kubernetes/sig-docs-es-owners , actually

**Comment by gavinfish** [comment]
/assign @raelga

It seems that alexbrand and raelga are the only owners.

https://github.com/kubernetes/org/blob/6e8e3e4a1cacb591a685fe84cb224c91b6b6c64b/config/kubernetes/sig-docs/teams.yaml#L61-L66

**Comment by gavinfish** [comment]
@sftim
Any help for this? Do you have the permission to make it merged?

- Enough lgtm from both de/es reviewers, approve from de owner
- The change is no difference from languages, also it will align with en version
- No active es owner for more than one month(I have assigned all possible owners)

**Comment by sftim** [comment]
> @sftim
> Any help for this? Do you have the permission to make it merged?

I can't approve changes for either German or Spanish localizations @gavinfish

**Comment by sftim** [comment]
You could try asking those localization teams via Slack?

(Here are links to the channels for [Spanish](https://app.slack.com/client/T09NY5SBT/CH7GB2E3B) & [German](https://app.slack.com/client/T09NY5SBT/CH4UJ2BAL))

**Comment by raelga** [comment]
Sorry for the delay folks 🙇

**Comment by gavinfish** [comment]
@sftim thanks for your suggestions.
@raelga thanks for your reviewing.

**Pull request merged**